### PR TITLE
Bug 1233533 - Fix intermittent test fail on Keyboard App.…

### PR DESCRIPTION
**Pull request by raylin**
… r=timdream

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e2f5456c7bb2ae2cb77c5cf37555e7d065e75e44

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6cc47fd1ef97d4bb71525c5377c9f25899fc807d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4ed36591e45039230030b351016eef1115af828e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9e8c3cee5e16cf3253d396003f4890bb05f4f407

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8d45b9ccf550d3f610abfaae33212eb59b1596d9

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f140c399b1c24f45ab5af560364d23c29dd44fcc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=19861b9442c0331cbf12ecf527c3a7e73eed9868

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=91a7efa7acbc08632e8f08336b06d30551e8f8a5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d2d1971fea14f0518661db8d2ca00eae38c4d784

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d70f1b921d6397553fe86c72b45cf8088452d725

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4d1b4f9dc628e4ac659f75f616afd86236af89c3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=973dd7c95e3ff987963af77b88af82049a6e55eb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2e38e81a4408045041bcc9484301838144aee7f7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f0099c5f4b2a7d42299da9b15003015d4d11ec7c

File: shared/test/integration/tbpl-manifest.json [hunk 1]
```diff
@@ -35,11 +35,7 @@
...
-    "apps/keyboard/test/marionette/auto_correction_test.js": "Bug 1235574 - Intermittent TEST-UNEXPECTED-FAIL | apps/keyboard/test/marionette/auto_correction_test.js",
-    "apps/keyboard/test/marionette/email_keyboard_test.js": "Bug 1233536 - Intermittent TEST-UNEXPECTED-FAIL | apps/keyboard/test/marionette/email_keyboard_test.js",
-    "apps/keyboard/test/marionette/number_keyboard_test.js": "Bug 1233533 - Intermittent TEST-UNEXPECTED-FAIL | apps/keyboard/test/marionette/number_keyboard_test.js",
-    "apps/keyboard/test/marionette/text_keyboard_test.js": "Bug 1235939 - Intermittent text_keyboard_test.js",
-    "apps/keyboard/test/marionette/uninstallation_test.js": "Bug 1237618 - Intermittent TEST-UNEXPECTED-FAIL | apps/keyboard/test/marionette/uninstallation_test.js",
+    "apps/homescreen/test/marionette/pinning_uninstall_test.js": "Bug 1233527 - Intermittent TEST-UNEXPECTED-FAIL | apps/homescreen/test/marionette/pinning_uninstall_test.js",
```
Comment (timdream) [inline shared/test/integration/tbpl-manifest.json]: rebase fail?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=16f47724835bf89ae5a73762fe465ddcabb0d85d

File: shared/test/integration/tbpl-manifest.json [hunk 2]
```diff
@@ -35,11 +35,6 @@
     "apps/email/test/marionette/send_imap_email_test.js": "Bug 975588 - Re-enable email marionette js tests",
     "apps/fm/test/marionette/fm_test.js": "Bug 1233777 - Intermittent TEST-UNEXPECTED-FAIL | apps/fm/test/marionette/fm_test.js | FM Radio test FM Radio favourites Test favouriting",
     "apps/homescreen/test/marionette/pinning_the_web_test.js": "Bug 1233529 - Intermittent TEST-UNEXPECTED-FAIL | apps/homescreen/test/marionette/pinning_the_web_test.js | Homescreen - Pin the web Pinning a site from the browser context menu",
```
Comment (raylin) [inline shared/test/integration/tbpl-manifest.json]: fix rebase fail

File: shared/test/integration/tbpl-manifest.json (hunk 1 above)
Comment (raylin) [inline shared/test/integration/tbpl-manifest.json]: fixed. Thanks

**Pull request merged**